client/components/deposits-status/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const DepositsEnabled = ( props ) => {
```

Comment:
I see that the component is called `DepositsEnabled`, but the file is `deposits-status` - can the component be renamed (or the file and the component name match)?